docs/source/models_and_datasets.md

```diff
@@ -17,4 +17,15 @@ limitations under the License.
 
 # Morpheus Models
 
-Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
\ No newline at end of file
+Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
```

Comment:
This link is good because its explicitly stated as a link to the latest version, but the rest should be relative links so that it matches the current version the user has.

Reply:
I alluded to it in the symlink discussion, but Sphinx doesn't like relative links in the docs (as far as I could tell). Is there a way to do the relative links that Sphinx won't balk at?